owasp-top10-2017-apps/a1/mongection/src/db.js

```diff
@@ -29,7 +29,7 @@ const login = async (credentials) => {
     try {
         const { email, password } = credentials;
 
-        const existsUser = await User.find({$and: [ { email: email}, { password: password} ]});
+        const existsUser = await User.find({$and: [ { email: email.toString()}, { password: password.toString()} ]});
```

Comment:
This simple solution doesn't handle `null`, the proper simple solution would have been to use `String(email)` and `String(password)` here.